server/opts_test.go

```diff
@@ -890,6 +890,94 @@ func TestNkeyUsersConfig(t *testing.T) {
 	}
 }
 
+// Test new nkey users
+func TestTlsWhitelist(t *testing.T) {
+	confFileName := createConfFile(t, []byte(`
+	tls {
+		cert_file: "./configs/certs/server.pem"
+		key_file: "./configs/certs/key.pem"
+		# Require a client certificate and map user id from certificate
+		verify: true
+		pinned_certs: ["7f83b1657ff1fc53b92dc18148a1d65dfc2d4b1fa3d677284addd200126d9069", 
+			"a8f407340dcc719864214b85ed96f98d16cbffa8f509d9fa4ca237b7bb3f9c32"]
+	}
+	cluster {
+		listen localhost:4223
```

Comment:
I know that you are not running the server here, so ports don't really matter, but probably use `-1`. I noticed that you have repeated `4224`.

Reply:
will fix